func_sig_registry/templates/base.html

```diff
@@ -16,7 +16,8 @@
         </div><!-- /.navbar-header -->	
         <div>
 					<ul class="nav navbar-nav">
-						<li><a href="{% url 'signature-list' %}">Browse Signatures</a></li>
+            <li><a href="{% url 'signature-list' %}">Browse Function Signatures</a></li>
+            <li><a href="{% url 'event-signature-list' %}">Browse Event Signatures</a></li>
```

Comment:
It would be nice to fix the mix of tabs and spaces here.  Your code is using spaces.  The existing code is using tabs.

Reply:
Turns out my IDE was not configured properly. Current change should fix this.